web/stockwatch-app/pages/leaderboard.js

```diff
@@ -1,81 +1,256 @@
-import clsx from 'clsx';
-import { Star } from 'phosphor-react';
-import styles from '../styles/leaderboard.module.scss';
+import { useEffect, useRef, useState } from 'react';
+import { useTable } from 'react-table';
+import { Button, Grid, Header, Input, Loader, Table } from 'semantic-ui-react';
+import { fetchAllTimeLeaderboard } from '../copied/RealityStockWatchBackend';
+import _ from 'lodash';
```

Comment:
Note: for cleanup, we might want to import methods directly instead of entire library